doc/whats_new/upcoming_changes/sklearn.manifold/31322.feature.rst

```diff
@@ -0,0 +1,3 @@
+- :class:`manifold.ClassicalMDS` was implemented to perform classical MDS
+  (eigendecomposition of the double-centered distance matrix).
+  By :user:`Dmitry Kobak <dkobak>`
```

Comment:
It could be nice to credit Micky774 for the work on the preceding PR.

Reply:
You are right. Added.